File: app/src/main/java/com/zeyad/rxredux/screens/user/list/events/DeleteUsersEvent.java
package com.zeyad.rxredux.screens.user.list.events;

import java.util.List;

import com.zeyad.rxredux.core.redux.BaseEvent;

/** @author by ZIaDo on 3/27/17. */
public final class DeleteUsersEvent implements BaseEvent {

    private final List<String> selectedItemsIds;

    public DeleteUsersEvent(List<String> selectedItemsIds) {
        this.selectedItemsIds = selectedItemsIds;
    }

    public List<String> getSelectedItemsIds() {
        return selectedItemsIds;
    }
}

File: app/src/main/java/com/zeyad/rxredux/screens/user/list/events/DeleteUsersEvent.kt
package com.zeyad.rxredux.screens.user.list.events

import com.zeyad.rxredux.core.BaseEvent
import kotlinx.android.parcel.Parcelize

@Parcelize
class DeleteUsersEvent(private val selectedItemsIds: List<String>) : BaseEvent<List<String>> {
    override fun getPayLoad(): List<String> = selectedItemsIds
}
